records/generator.go

```diff
@@ -232,24 +258,40 @@ func (rg *RecordGenerator) InsertState(sj StateJSON, domain string, ns string,
 				continue
 			}
 
-			tname := labels.AsDNS952(task.Name)
-			sid := slaveIdTail(task.SlaveId)
-			tag := hashString(task.Id)
-			tail := fname + "." + domain + "."
+			fields := map[string]string{
+				"sid": slaveIdTail(task.SlaveId),
```

Comment:
This is not the final word yet. Next step is to identify the values we want in the patterns, now with the limited knowledge of the tasks, later from the `DiscoveryInfo` fields.